node/cli/src/chain_spec.rs

```diff
@@ -132,6 +132,7 @@ fn cord_development_config_genesis() -> serde_json::Value {
 			get_account_id_from_seed::<sr25519::Public>("Alice"),
 		)],
 		get_account_id_from_seed::<sr25519::Public>("Alice"),
+		29,
```

Comment:
Chain-spec will not allow the creation of a new variable at the top level. The only place to insert a new variable is via pallet config.

Reply:
@smohan-dw would like to understand this a bit more. I was able to test getting the prefix loaded up in genesis_build call in pallet through this logic.

`2024-03-18 08:08:37.151  INFO main cord_identifier::pallet: Setting new prefix key 29`

(this log came with latest PR, which was done just to highlight passing of values from chain_spec perspective).